nodejs/src/handlers/list_available_badges.ts

```diff
@@ -0,0 +1,13 @@
+
+/** 
+  * List badges that a user is able to Mint on a given platform.
+  * 
+  * Currently just logs the event.
+  * 
+  * @param: event - TODO
+  * @param: context - Lambda execution context. Not needed.
+*/
+exports.list_available_badges_handler = async function(event, context) {
```

Comment:
seems to be a copy of `list_badges`

Reply:
It is - this is just a template. Will fill in code later.